scss/_base_icon-definitions.scss

```diff
@@ -534,3 +534,16 @@ $social-icon-size: map-get($icon-sizes, heading-icon--small);
 
   display: inline-block;
 }
+
+%leading-linux-prompt-icon {
```

Comment:
It's quite specific to the code components, so I'd expect to find in with base code styles I think.